crates/miden-objects/src/note/note_type.rs

```diff
@@ -113,3 +128,38 @@ impl Deserializable for NoteType {
         Ok(note_type)
     }
 }
+
+// DISPLAY
+// ================================================================================================
+
+impl Display for NoteType {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            NoteType::Private => write!(f, "Private"),
+            NoteType::Encrypted => write!(f, "Encrypted"),
+            NoteType::Public => write!(f, "Public"),
```

Comment:
```suggestion
            NoteType::Private => write!(f, "private"),
            NoteType::Encrypted => write!(f, "encrypted"),
            NoteType::Public => write!(f, "public"),
```

These should be lower case then.